migrations/V83__Add_EUA_to_508_Status_Records.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE accessibility_request_status_records ADD COLUMN eua_user_id TEXT CHECK (eua_user_id ~ '^[A-Z0-9]{4}$');
```

Comment:
`(eua_user_id ~ '^[A-Z0-9]{4}$')` will return null if no match is found, so having a null value in `eua_user_id` will not fail this check constraint.